README.md

```diff
@@ -176,3 +178,11 @@ bots. If you are a beginner developer, please use this boilerplate.
 
 - There is a step by step walkthrough to learn how to create Discord bots with
   Discordeno on our website! https://discordeno.mod.land/stepbystep
+
+---
+
+## Troubleshooting
+
+### `Module “./deps” has no exported member X deno-ts(2305)`
+
+This is most likely because you don’t have the modules cached. Run `deno cache mod.ts` and then reload your code editor.
```

Comment:
Can you remove this section entirely? We no longer use a `deps.ts` file...

Reply:
lol